server.js

```diff
@@ -1,35 +1,28 @@
 import express from "express";
+import {} from "dotenv/config";
 import cors from "cors";
-import mongoose from "mongoose";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
-mongoose.Promise = Promise;
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+import connectDB from "./config/db";
+import routes from "./routes/restaurantRoutes";
+import { errorHandler } from "./middlewear/errorMiddlewear";
+
+
+const swaggerUi = require("swagger-ui-express"),
```

Comment:
Great job using Swagger for this project! It was really interesting and inspiring to see how you implemented it.